Packages/Nakama/Runtime/UnityWebRequestAdapter.cs

```diff
@@ -123,6 +123,14 @@ private UnityWebRequestAdapter()
             }
             else if (IsHttpError(www))
             {
+                if (www.responseCode >= 500)
+                {
+                    // TODO think of best way to map HTTP code to GRPC code since we can't rely
+                    // on server to process it. Manually adding the mapping to SDK seems brittle.
+                    errback(new ApiResponseException((int) www.responseCode, "", -1));

```

Comment:
@lugehorsam I think we could dump the load balancer HTML response in this scenario into the exception message. What do you think?

Reply:
@novabyte sure makes sense 👍

Reply:
done